### PR TITLE
Resources table

### PR DESCRIPTION
**Pull request by cleferman**
### What does it fix?

Closes #XXX

### CODE REVIEW

File: components/ResourcesTableList/ResourceTableItem.tsx [hunk 1]
```diff
@@ -20,8 +22,10 @@ const ResourceTableItem = ({ name, quantity, um }: ResourceTableItemProps) => {
     >
       <div className="w-50%">{name}</div>
       <div className="w-50% flex items-center">
-        <span className="mr-3">{`${quantity} ${um}`}</span>
-        <Image alt="delete_icon" src="/icons/delete.svg" />
+        {quantity && um && <span className="mr-3">{`${quantity} ${um}`}</span>}
+        <button onClick={() => onDelete && onDelete(id)}>
+          <Image alt="delete_icon" src="/icons/delete.svg" />
+        </button>
```
Comment (rhadu) [inline components/ResourcesTableList/ResourceTableItem.tsx:28]: shouldn't this be hidden if there's no `onDelete` function? 
``` js
{ onDelete  && (
<button onClick={() => onDelete(id)}>
  <Image alt="delete_icon" src="/icons/delete.svg" />
</button>
)}

File: components/SignUpProducts/GenericProduct.tsx [hunk 2]
```diff
@@ -12,22 +12,24 @@ interface IProps {
...
   const {
     handleSubmit,
     register,
     formState: { errors },
     control,
   } = useForm()
 
-  const onSubmit = (values: any) => {
+  const onFormSubmit = (values: any) => {
     console.log('values', { ...values, category })
```
Comment (rhadu) [inline components/SignUpProducts/GenericProduct.tsx:27]: remove

File: public/locales/ro/common.json [hunk 3]
```diff
@@ -165,5 +165,9 @@
   "error.county.minOne": "Trebuie să selectați minim un județ",
   "controls.counties.selected": "județe selectate",
   "gdpr.consent": "Prin completarea formularului accept ca datele mele să fie procesate de Departamentul pentru Situații de Urgență. Cunosc faptul că Departamentul pentru Situații de Urgență prelucrează datele mele cu caracter personal în baza prevederilor Regulamentului (UE) 2016/679.",
-  "signup.error.gdpr.required": "Pentru a merge mai departe trebuie să vă dați acordul cu privire la procesarea datelor cu caracter personal"
+  "signup.error.gdpr.required": "Pentru a merge mai departe trebuie să vă dați acordul cu privire la procesarea datelor cu caracter personal",
+  "error.availability.required": "Trebuie să selectați o valoare",
+  "resources.product": "Produs",
+  "resources.quantity": "Cantitate",
+  "resources.added.products": "Produse adăugate"
```
Comment (rhadu) [inline public/locales/ro/common.json:172]: when fixing comments add them please at the end of the `common.json`

Comment (cleferman) [reply public/locales/ro/common.json]: are you referring to `services.driver-name`?

**Review by rhadu — APPROVED** [review]

**Pull request merged**